README.md

```diff
@@ -66,121 +67,80 @@ This approach dramatically reduces latency and reduces load on the OP's servers.
 
-Here is are examples: 
+- vp_ldp:  A claim whose value is a W3C Verifiable Presentation object using the JSON-LD representation, which is a JSON object.  The claim’s value may also be an array of W3C Verifiable Presentation objects using the JSON-LD representation if the use case calls for multiple JSON-LD VPs.
 
-```json
-{
-   "id_token":{
-      "acr":null,
-      "vp":{
-         "credential_types":[
-            "https://www.w3.org/2018/credentials/examples/v1/IDCardCredential"
-         ]
-      }
-   }
-}
-```
+Note that above claims have to be distinguished from `vp` or `vc` claims as defined in [JWT proof format](https://www.w3.org/TR/vc-data-model/#json-web-token). `vp` or `vc` claims contain those parts of the standard verifiable credentials and verifiable presentations where no explicit encoding rules for JWT exist. They are not meant to include complete verifiable credentials or verifiable presentations objects which is the purpose of the four claims defined in this specification.
 
-### vc in id_token
+This table shows the different combinations of covered by the claims defined in this specificaiton.
 
-A Verifiable Credential embedded in an ID Token is requested by adding a element `vc` to the `id_token` top level element of the `claims` parameter. This element must contain a `credential_types` sub element as defined above.
+|  | vc_jwt | vp_jwt | vc_ldp | vp_ldp
+|:----------------|:---------------|:---------------|:---------|:--------------------------|
+| Object included in the claim | verifiable credential | verifiable presentation | verifiable credential | verifiable presentation 
+| Proof format on the object| JWT | JWT | LD-Proof | LD-Proof
 
-### vp_token
+There are multiple candidates for requesting verifiable presentations and verifiable credentials using OpenID Connect flows: Edmund's Aggregated Claims draft, DIF Presentation Exchange, below draft, and probably others. This would be a natural next step after defining claims, and agreeing on the request syntax should be separate from agreeing on the usage of the above four claims.
 
-A VP Token is requested by adding a new top level element `vp_token` to the `claims` parameter. This element contains the following sub elements:
 
-`credential_types`
-Object array containing definitions of credential types the RP wants to obtain along with an (optional) definition of the claims from the respective credential type the RP is requesting. Each of those object has the following fields:
+## Requesting Verifiable Presentations
 
-* `type` - String denoting a credential type
+A RP requests a Verifiable Presentation using the `claims` parameter. 
 
-* `claims` - An object determining the claims the RP wants to obtain using the same notation as used underneath `id_token`. 
+### Verifiable Presentation object in id_token
```

Comment:
the section is still just a small part of what exists in the main revision. 

For example, it misses the ability to request a subset of claims.